ci-operator/step-registry/rosa/cluster/credentials/hypershift/mgmt/rosa-cluster-credentials-hypershift-mgmt-commands.sh

```
@@ -30,15 +30,16 @@ else
 fi
 
 # Get the kubeconfig of the management cluster who manages the hosted cluster
-echo "Get the kubeconfig of the manangement cluster ..."
+echo "Get the kubeconfig of the management cluster ..."
 HOSTED_CLUSTER_ID=$(cat "${SHARED_DIR}/cluster-id")
 MC_NAME=$(ocm get /api/clusters_mgmt/v1/clusters/${HOSTED_CLUSTER_ID}/provision_shard | jq -r .management_cluster)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)